rel/config/config.exs

```diff
@@ -111,6 +111,11 @@ if Utils.Environment.exists?("BASIC_AUTH_USERNAME") do
     ]
 end
 
+# Configure CORS
+config :elixir_boilerplate, :corsica,
+  origins: System.get_env("CORS_ALLOWED_ORIGINS"),
```

Comment:
What is `CORS_ALLOWED_ORIGINS` exactly? A comma-separated list of origins? Eg. `"foo.com,bar.com"`?

Reply:
Good question, for now it’s a single string but we should probablement split it if we want to support a comma-separated list of origins as you mentioned.

Reply:
I just looked at the [Origins](https://hexdocs.pm/corsica/Corsica.html#module-origins) section and for multiple origins it’s:

```elixir
plug Corsica, origins: ["http://foo.com", "http://bar.com"]
```

This is not possible with the current setup, is it?

Reply:
```suggestion
  origins: String.split(System.get_env("CORS_ALLOWED_ORIGINS"), ","),
```

And I’m guessing `["*"]` will work 👍

Reply:
We should probably refrain from using `System.get_env` like the rest of the file. We could have a `Utils.Environment.get_list("SENTRY_DSN")` which would do the heavy lifting for us 😄

Reply:
Yes, I’m gonna wait for PR #35 to be merged and will complete the setup afterward.